setup.cfg

```diff
@@ -31,7 +31,7 @@ install_requires =
     scipy>=1.0
     packaging
 setup_requires =
-    numpy>=1.16.6,<1.20
+    numpy>=1.13.3
     scipy>=1.0
     cython>=0.29.20
     packaging
```

Comment:
Don't you just want to use `oldest-supported-numpy` again?  (`setup_requires` is pretty unnecessary anyway - it's useful as a last resort check, but with pip's `--no-build-isolation`, we'll probably be able to do away with all our direct access to `setup.py`)

Reply:
I think this is correct-ish as it is. If I specify oldest supported numpy, then I'm worried calling setup.py will attempt to install the oldest support numpy, which is not what one wants in many cases (e.g. one is likely trying to build with the numpy one already has). Perhaps I should just remove the restriction entirely and write just "numpy"?

Reply:
Yeah, that's a good point. Thanks!